.github/workflows/claim-crates.yml

```diff
@@ -0,0 +1,28 @@
+name: Claim Crates
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  claim-crates:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@3cf7f8cc28d1b4e7d01e3783be10a97d55d483c8 # v2.7.1
+        with:
+          cache-on-failure: true
+
+      - name: install parity-publish
+        run: cargo install parity-publish
```

Comment:
can we pin the crate version?

Reply:
Any reason? It's a crate controlled by us so we push updates as needed. But I can pin it if that's wanted

Reply:
Pinned now.